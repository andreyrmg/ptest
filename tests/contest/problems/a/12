30
35 23
25 1E
4 1120
21 40
34 F
33 11
20 2A
14 2B
5 11
26 2B
30 1F
12 8
36 26
14 61
9 21
20 3F
5 22
23 20
10 18
3 211
12 72
31 J
20 4F
30 35
2 1011001
16 5
4 20
9 28
22 2H
29 R
