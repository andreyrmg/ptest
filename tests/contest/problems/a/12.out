16 5
5 11
4 20
12 8
5 22
34 F
10 18
9 21
31 J
3 211
9 28
29 R
33 11
14 2B
25 1E
30 1F
23 20
20 2A
22 2H
26 2B
35 23
20 3F
36 26
21 40
14 61
12 72
4 1120
2 1011001
20 4F
30 35
